docs/terraform/overview.md

```diff
@@ -17,7 +17,7 @@ You can continue working with CPLN configuration files in YAML format and start
 
 ## Usage
 
-Let's take a look at how to deploy a [simple Rails application](example/.controlplane/controlplane.yml) on CPLN using Terraform:
+Let's take a look at how to deploy a [simple Rails application](https://github.com/shakacode/control-plane-flow/tree/main/docs/terraform/example/.controlplane/controlplane.yml) on CPLN using Terraform:
```

Comment:
I think we need relative links for https://www.shakacode.com/control-plane-flow/docs/.

What inspired this PR?

Reply:
Inspiration came from #246